eurovision/notebooks/story.ipynb

```diff
@@ -1053,7 +1103,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Average deviations"
+    "From this plot, we can see a few patterns emerging. Some countries tend to vote higher for each other, forming lighter squares in the heatmap. One such square in the bottom right is around slovenia, bosnia and herzegovina, croatia, serbia, which all give each other relatively high votes on average. But some countries will just genuinely perform well. To disentangle these two factors, we can visualise how each country deviates from the average vote for another country. The next heatmap takes a bit more preparation:"
```

Comment:
No more flags? There really nice, but I also understand that maybe it gets too messy to continue including them. But I think probably it's good to capitalise the country names.

Reply:
Forgot! Put them in now :)